app/models/spree_avatax/config.rb

```diff
@@ -11,6 +13,17 @@ class << self
       attr_accessor :sales_invoice_generate_error_handler
       attr_accessor :sales_invoice_commit_error_handler
       attr_accessor :sales_invoice_cancel_error_handler
+
+      def timeout
+        (config = last) ? config.timeout : DEFAULT_TIMEOUT
```

Comment:
It makes me just a tad uncomfortable to use `last` without an `order`. I know that this will order by id by default, but personally I'd love to be more explicit about this. Not a blocker, just a thought.